Spec/RealtimeClient.swift

```diff
@@ -1328,6 +1328,56 @@ class RealtimeClient: QuickSpec {
                 expect(result).to(equal(expectedOrder))
             }
 
+            // Possible issue of https://github.com/ably/ably-ios/issues/640
+            xit("should unlock queue when the timer ends") {
```

Comment:
I think we can remove this one, the other reproduces the issue more clearly from a user persoective without meddling with the internal emitter.